app/lib/features/home/widgets/sidebar_widget.dart

```diff
@@ -61,7 +61,7 @@ class SidebarWidget extends ConsumerWidget {
               key: Keys.avatar,
               margin: const EdgeInsets.only(top: 8),
               child: InkWell(
-                onTap: () => context.pushNamed(Routes.myProfile.name),
+                onTap: () => context.goNamed(Routes.settings.name),
```

Comment:
You sure this wouldn't break the routing part ? Need to be careful thinking about changing route transition type...